pkgs/misc/meson64-tools/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, stdenv, fetchFromGitHub, buildPackages }:
+
+stdenv.mkDerivation rec {
+  pname = "meson64-tools";
+  version = "0.0";
```

Comment:
This should be some form of date based version, although the exact format is subject to some debate: https://github.com/NixOS/nixpkgs/pull/100833. I'd suggest `unstable-YYYY-MM-DD` since that is the most common and it will just become part of the eventual treewide change.

Reply:
Yeah, I was curious about it when I was typing "0.0". Thanks